File: llm_written/claude-sonnet-4.5/bfs/c#/Graph.cs
using System.Collections.Generic;

public class Graph
{
    public Dictionary<int, List<int>> Vertices { get; } = new Dictionary<int, List<int>>();

    public void AddEdge(int fromVertex, int toVertex)
    {
        if (!Vertices.ContainsKey(fromVertex))
            Vertices[fromVertex] = new List<int>();
        if (!Vertices.ContainsKey(toVertex))
            Vertices[toVertex] = new List<int>();
        
        Vertices[fromVertex].Add(toVertex);
        Vertices[toVertex].Add(fromVertex);
    }
}

File: llm_written/claude-sonnet-4.5/bfs/go/graph.go
package main

import "sort"

// Graph represents an undirected graph using an adjacency list
type Graph struct {
	Vertices map[int][]int
}

// AddEdge adds an undirected edge between two vertices
func (g *Graph) AddEdge(from, to int) {
	if g.Vertices == nil {
		g.Vertices = make(map[int][]int)
	}
	g.Vertices[from] = append(g.Vertices[from], to)
	g.Vertices[to] = append(g.Vertices[to], from)
}

// Normalize ensures deterministic iteration by sorting all adjacency lists
func (g *Graph) Normalize() {
	for vertex := range g.Vertices {
		sort.Ints(g.Vertices[vertex])
	}
}
